### PR TITLE
enh:configurable parameters

### PR DESCRIPTION
**Pull request by sanuann**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -395,6 +396,9 @@ export default {
     }
   },
   mounted() {
+    axios.get('https://sig.mit.edu/vb/').then((response) => {
```
Comment (satra) [inline src/App.vue:399]: make this configurable too. so that people can point to their own server.

File: src/config.js [hunk 2]
```diff
@@ -1,5 +1,8 @@
 export default {
   /* eslint-disable */
   githubSrc: 'https://raw.githubusercontent.com/ReproNim/reproschema/master/protocols/VoicePilot/VoicePilot_schema',
-  contentSrc: 'https://raw.githubusercontent.com/ReproNim/reproschema/master/protocols/VoicePilot/README.md'
+  contentSrc: 'https://raw.githubusercontent.com/ReproNim/reproschema/master/protocols/VoicePilot/README.md',
```
Comment (satra) [inline src/config.js:4]: is contentSrc used somewhere, since we have landingpage in the protocol now?

File: src/App.vue [hunk 3]
```diff
@@ -395,6 +396,9 @@ export default {
...
+      console.log(399, response);
```
Comment (satra) [inline src/App.vue:400]: should there be a debug flag that logging is dependent on?

File: src/config.js (hunk 2 above)
Comment (sanuann) [reply src/config.js:4]: if in case a protocol doesn't have landingPage, we can perhaps use contentSrc to point to some generic landing page. What say?

Comment (satra) [reply src/config.js:4]: i wouldn't do this. the protocol must have a landing page.

Comment (satra) [reply src/config.js:4]: if it doesn't the landing page will be empty and only show the button.

Comment (sanuann) [reply src/config.js:4]: ok, will update.

File: src/App.vue (hunk 3 above)
Comment (satra) [reply src/App.vue:400]: added a few issue #149

**Pull request merged**